makefile

```diff
@@ -73,15 +73,15 @@ INCLUDE_PATHS = \
 	-Istdx-allocator/source \
 	-Idmd/src
 
-DMD_VERSIONS = -version=StdLoggerDisableWarning -version=CallbackAPI -version=DMDLIB
+DMD_VERSIONS = -version=StdLoggerDisableWarning -version=CallbackAPI -version=DMDLIB -version=MARS
```

Comment:
This should probably reflected in build.bat also

Reply:
This is needed in order to solve __VENDOR__ macro. If we don't include version=MARS, the current version of the parser can't parse this macro